css.md

```diff
@@ -137,3 +138,15 @@ $_componentNameColor: #000;
 ```scss
 $_componentNameColor: $color1;
 ```
+
+### [2.3](#scss--comments): Comments
+
+When commenting in sass be sure to use `//` version of comments this version will be stripped out during the compilation procress
+
+```scss
+//Bad
+$_componentNameColor: #000; /*this is a bad comment */
+
+//Good
+$_componentNameColor: $color1; //this the right way to comment
```

Comment:
technically, the comment should be on a separate line

Reply:
Fixed in latest checkin to branch